src/tests/ftest/pool/target_query.yaml

```
@@ -0,0 +1,90 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+  test_clients:
+    - client-A
+timeout: 800
+setup:
+  start_agents_once: False
+  start_servers_once: False
+server_config:
+  engines_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["aaaa:aa:aa.a"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+      targets: 2
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31517
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["bbbb:bb:bb.b"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+      targets: 2
+pool:
+    mode: 146
+    name: daos_server
+    scm_size: 2G
+    nvme_size: 16G
+    control_method: dmg
+    pool_query_timeout: 30
+container:
+    type: POSIX
+    control_method: daos
+ior:
+  api: "DFS"
+  client_processes:
+   np: 1
+  dfs_destroy: False
+  iorflags:
+   flags: "-w -F -k -G 1 -vv"
```

Minor spacing - This will eventually be enforced by yaml linting
```suggestion
  client_processes:
    np: 1
  dfs_destroy: False
  iorflags:
    flags: "-w -F -k -G 1 -vv"
```

---

Done